docs/content/1.getting-started/1.index.md

```diff
@@ -102,11 +102,15 @@ variant: subtle
 ---
 ::
 
-#### `CACHE_CLEANUP_OLDER_THAN_DAYS`
+#### `CACHE_CLEANUP_UNTOUCHED_TTL_DAYS`
 
-- Default: `90`
+- Default: `30`
 
-The number of days to keep stale cache data and metadata before deleting it. Set to `0` to disable cache cleanup.
+Cache entries which have not been accessed for `CACHE_CLEANUP_UNTOUCHED_TTL_DAYS` days will be automatically deleted. Set to `0` to disable.
+
+#### `CACHE_CLEANUP_TTL_DAYS`
```

Comment:
default?

Reply:
not set by default

Reply:
Maybe add as to documentation: Disabled if not set (default).